vinyldns.go

```diff
@@ -142,6 +142,58 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:        "zone-create",
+			Usage:       "zone-create --name <name> --email <email> --admin-group-id <adminGroupID> --transfer-connection-name <transferConnectionName> --transfer-connection-key <transferConnectionKey> --transfer-connection-key-name <transferConnectionKeyName> --transfer-connection-primary-server <transferConnectionPrimaryServer> --zone-connection-name <zoneConnectionName> --zone-connection-key <zoneConnectionKey> --zone-connection-key-name <zoneConnectionKeyName> --zone-connection-primary-server <zoneConnectionPrimaryServer>",
+			Description: "Create a zone",
+			Action:      zoneCreate,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "name",
+					Usage: "The zone name",
+				},
+				cli.StringFlag{
+					Name:  "email",
+					Usage: "The zone email",
+				},
+				cli.StringFlag{
```

Comment:
would it be easier from a UX perspective to take an admin group name instead of id?

Reply:
Yes, arguably. Will the API accept that, or would that UX require I...
1. call the API to find the group ID of the group with the provided name
2. use the group ID from step #1 in the zone creation call
?

Reply:
You would have to the steps you mentioned.

It also provides some nice communication.  If the caller enters a name for a group they do not belong to, we can predictably fail with a nice error message :)

Reply:
I like that, though I'm inclined to suggest we proceed in using `id` for now, as we do elsewhere for other resource types. Then, we can roll out a `--admin-group-name` flag in the future. If you agree, I can create an issue to do so.

Reply:
@mdb sounds good